win_images/win_packaging.ps1

```diff
@@ -25,10 +49,16 @@ Set-ExecutionPolicy Bypass -Scope Process -Force
 [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072
 iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))
 
-# Install Git, BZ2 archive support, Go, and the MingW (GCC for Win) compiler for CGO support
-# Add pstools to workaorund sess 0 WSL bug
-choco install -y git mingw archiver psexec; CheckExit
-choco install golang --version 1.19.2 -y; CheckExit
+# Install basic required tooling.
+#   psexec needed to workaround session 0 WSL bug
+#   mingw version 11.2, later versions may be incompatible with CGO on some versions of golang
+retryInstall git archiver psexec golang mingw@11.2.0; CheckExit
+
+# Update service is required for dotnet 3.5 (dep of wix)
+Set-Service -Name wuauserv -StartupType "Manual"; CheckExit
+
+# Install wixtoolset (and .Net) for installer build & test.
+retryInstall wixtoolset; CheckExit
```

Comment:
@n1hility when you have a sec.  I'm trying to move the mingw/wix install from podman CI to here, to try and save 5-10 minutes of podman-developer time.  Now I'm wondering if you had it setup that way for a reason?

The .Net install [seems to be breaking here fairly hard](https://cirrus-ci.com/task/5559138445426688).  If it's an easy fix, I'm eager to know, otherwise I'll just move it back into podman.  TIA.

Reply:
@cevich sorry I missed this message. I'm not sure why the package is failing. I would try two things, one putting the Set-Service higher before any installs, and the second thing is just only installing wixtoolset

Reply:
No worries, you're timing is fine.  This PR is a "nice to have" at this point, definitely not absolutely required.

I believe I tried both those suggestions with similar results, but I can try moving again.  The other thought I had is: Maybe a reboot is needed?  That would certainly explain how/why it works fine in Podman CI.